src/i18n/fr/translation.json

```diff
@@ -346,6 +346,7 @@
     "footer": {
       "product": "Mindlogger est un produit de",
       "support": "Soutien",
+      "privacy": "Confidentialité",
       "termsOfService": "Conditions d'utilisation"
```

Comment:
should this be `Conditions` instead?

Reply:
Ah, I thought this was a simple enough change that it could be addressed in this same PR. Ok